girderformindlogger/web_client/src/templates/body/userAccount.pug

```diff
@@ -26,7 +26,7 @@ ul.g-account-tabs.nav.nav-tabs
       form#g-user-info-form(role="form")
         .form-group
           label(for="g-login") Login
-          p.form-control-static= user.get('login')
+          input#g-login.input-sm.form-control(type="text", value=user.get('login'))
```

Comment:
This is good but unnecessary. This interface is vestigial from the upstream Girder project and not one in use for MindLogger.